resources/linux/rpm/code.spec.template

```diff
@@ -56,17 +55,14 @@ ln -sf /usr/share/@@NAME@@/bin/@@NAME@@ %{_bindir}/@@NAME@@
 update-mime-database /usr/share/mime &> /dev/null || :
 
 %postun
-if [ $1 = 0 ]; then
```

Comment:
I'm wondering why this section is being removed?

Reply:
Since the symlink will be part of the RPM payload, there's no need to do maintenance on it in the post-transaction scripts. It'll just be one of the files installed/removed during the actual transaction.